client/lib/domains/dns/index.js

```diff
@@ -45,8 +46,8 @@ function isValidDomain( name ) {
 	return /^([a-z0-9-_]{1,63}\.)*[a-z0-9-]{1,63}\.[a-z]{2,63}$/i.test( name );
 }
 
-function isValidName( name, type ) {
-	if ( isNameEmptyAndItsAllowed( name, type ) ) {
+function isValidName( name, type, domainName ) {
+	if ( isRootDomainAndItsAllowed( name, type, domainName ) ) {
```

Comment:
It's allowed - as in:
![214914](https://cloud.githubusercontent.com/assets/3392497/13216450/1d70254e-d95c-11e5-91dd-d9d68bd527de.gif)
Sorry, couldn't help myself :laughing: 

As in - the opposite of "forbidden". I tried making it read as close to normal language as possible - "if it is a root domain and it's allowed" (aka. "not forbidden"). I honestly can't think of a better word here :/